template-test/ci-run.sh

```diff
@@ -11,9 +11,15 @@ VARIANT="${VARIANT:-basic}"
 APP_NAME="${VARIANT}-test-app"
 
 rm -rf template-test/dummy/$APP_NAME
-cd template-test/dummy
+mkdir -p template-test/dummy/$APP_NAME
 
-echo -e $GENERATOR_INPUT | RACK_ENV=development RAILS_ENV=development rails new $APP_NAME -d postgresql -m $TEMPLATE
+# otherwise rails webpacker:install initiates yarn init and that overrides these yarn.lock package.json files 
+# exist under the root directory instead of creating new ones under template-test/dummy/$APP_NAME
+cd template-test/dummy/$APP_NAME && yarn init --yes
+cd ..
```

Comment:
```suggestion
( cd template-test/dummy/$APP_NAME && yarn init --yes )
```

Wrapping in `()` means we run in a subshell, so `cd`ing will be scoped to that shell - it's a bit nicer because it's one less line and if the `cd` fails for some reason (unlikely, but not impossible) we won't `cd` out of the root directory.

Reply:
Ah yes, thanks :)